### ISSUE TITLE
seconde in log format.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10848]
|<img alt="stilly's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| stilly reported|
|-|-|
|Trac ID|trac#1104|
|Type|enhancement|
|Created|2005-07-06 16:15:47Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1104 1104
type__enhancement enhancement
reporter__stilly stilly
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1120666547000000 1120666547000000
changetime__1170347040000000 1170347040000000
version__ 
owner__ 
cc__stilly
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="stilly's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|stilly commented|
|-|-|

```
#!html
<pre>
Is it possible to add seconde in log format

file twisted/python/log.py : class FileLogObserver
timeformat = "%Y/%m/%d %H:%M %Z" --> "%Y/%m/%d %H:%M:%S %Z"
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set status to `closed`|
|-|-|

Closed with [#10566](https://github.com/twisted/twisted/issues/10566).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
